src/svg/graphic.ts

```diff
@@ -69,6 +70,14 @@ function setStyleAttrs(attrs: SVGVNodeAttrs, style: AllStyleOption, el: Path | T
     setShadow(el, attrs, scope);
 }
 
+function setMetaData(attrs: SVGVNodeAttrs, el: Path | TSpan | ZRImage) {
+    if (el.__metaData) {
+        for (const key in el.__metaData) {
+            attrs[META_DATA_PREFIX + key] = el.__metaData[key] + '';
```

Comment:
Perhaps we should convert the attribute key to lower case explicitly.
See https://stackoverflow.com/a/25033330/6075217